### PR TITLE
Fixes: #462 (partially) Add checkbox instead of radio button group

### PR DESCRIPTION
**Pull request by isuruAb**
Fixes issue #462 Partially

Changes: Remove radio button group and replace it with checkbox

Demo Link: http://isuruab.surge.sh/

Screenshots for the change: 

screen shot 2017-07-19 at 8 42 24 am

@rishiraj824  @madhavrathi  @uday96  please review this

### CODE REVIEW

**Comment by uday96** [comment]
@isuruAb This looks neat :+1: But @mariobehling mentioned that : 
>Take out "Standard Server" and only keep custom server. Make the custom server button into a button that can be activate or deactivated. When it is activated it should show URL field in the place of the the text "Custom Server"

Please confirm once as to which method should be followed

**Comment by isuruAb** [comment]
@uday96  can you check this again

**Comment by mariobehling** [comment]
>  When it is activated it should show URL field in the place of the the text "Custom Server"

When the field is activated in the place of "custom server" show the field to add the custom URL!

**Comment by uday96** [comment]
su

@isuruAb There is a slight overlap between error text and sign up button. Please check. Rest looks good :+1:

**Comment by mariobehling** [comment]
Open the signup page and click on the movable button. Result in my view: There is a "jump", that should not be there.

Please increase the distance between "custom server" and "sign up" to be more natural.
screenshot from 2017-07-19 08-21-24

**Comment by mariobehling** [comment]
Could you make the field that shows up where to enter the "URL" exactly in the same position as "Use Custom Server" please?

File: src/components/Auth/SignUp/SignUp.react.js [hunk 1]
```diff
@@ -280,24 +295,22 @@ export default class SignUp extends Component {
...
     render() {
-
-        const serverURL = <TextField name="serverUrl"
-                            onChange={this.handleChange}
+        const customUrlStyle= {
+            width:'202px',
+            textAlign:'left',
+            margin:'-35px 0 0px 5px',
+        }
+        const serverURL = <TextField
```
Comment (rishiraj824) [inline src/components/Auth/SignUp/SignUp.react.js]: @isuruAb Add a onClick function and handle the this.state.checked state there to achieve the effect.

**Comment by mariobehling** [comment]
Good job! Would be great, if you could also do this for the login and settings.

**Pull request merged**